Tools/autotest/common.py

```diff
@@ -6557,11 +6557,18 @@ def get_stacktrace(self):
 
     def get_exception_stacktrace(self, e):
         if sys.version_info[0] >= 3:
+            if sys.version_info[1] >= 10:
```

Comment:
much simpler to just do this:
https://github.com/ArduPilot/MAVProxy/pull/1020/files

Reply:
I'd gone with the new signature on the basis that we won't have to come back in Python 3.12 and fix our code *again* when they remove the compatability later.

Still, may as well be consistent and I don't care that much; I've made this use the simpler patch.